tests/test_inverse.py

```diff
@@ -249,15 +250,6 @@
     )
 )
 
-TESTS.append(
```

Comment:
@rijobro I removed this test case here because it's duplicated with previous lines, seems like a copy typo.

Thanks.